translations/ui-orders/en.json

```diff
@@ -194,7 +194,7 @@
   "errors.physicalLocCostQtyMismatch": "PO line physical quantity and locations physical quantity do not match",
   "errors.piecesNeedToBeCreated": "Pieces need to be created",
   "errors.polLimitExceeded": "Your FOLIO system is configured to limit the number of PO Lines on each order",
-  "errors.poLineHasRelatedApprovedInvoice": "The purchase order line refers to an approved invoice in the current fiscal year",
+  "errors.poLineHasRelatedApprovedInvoice": "The purchase order line fund distribution can not be change because the order line is linked to an invoice line that is currently in the status approved.",
```

Comment:
```suggestion
  "errors.poLineHasRelatedApprovedInvoice": "The purchase order line fund distribution can not be changed because the order line is linked to an invoice line that currently has the "approved" status.",
```

Reply:
Adjustment to the language of the message.

Reply:
Thank you John! I have to first agree on the text with PO before commit it. 🙂